lib/fragment/structured_text/integration_fields.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Prismic.Fragment.IntegrationFields do
```

Comment:
I think this file should not be inside the `structured_text` folder as it's only a fragment.